### PR TITLE
Join AD

### PR DESCRIPTION
**Pull request by jpnurmi**
See:
- https://github.com/canonical/subiquity/pull/1572

### CODE REVIEW

File: lib/src/client.dart [hunk 1]
```diff
@@ -516,6 +516,14 @@ class SubiquityClient {
       AdPasswordValidation.values.byName,
     );
   }
+
+  Future<AdJoinResult> joinActiveDirectory({bool wait = true}) async {
```
Comment (CarlosNihelton) [inline lib/src/client.dart:520]: Don't you feel like that method name doesn't convey an idea of taking an action?

Comment (jpnurmi) [reply lib/src/client.dart:520]: how would you prefer to name it? :) `getActiveDirectoryJoinResult()`?

Comment (CarlosNihelton) [reply lib/src/client.dart:520]: Yeap! Sounds better. We are just monitoring a result of an action initiated by somebody else.

**Review by CarlosNihelton — APPROVED** [review]
LGTM :+1:

**Pull request merged**